colorizer_data/converter.py

```diff
@@ -134,13 +145,25 @@ def _write_data(
     writer: ColorizerDatasetWriter,
     config: ConverterConfig,
 ):
+    outliers_data = _get_data_or_none(dataset, config.outlier_column)
+    if outliers_data is not None:
+        outliers_data = outliers_data.astype(bool)
+        if outliers_data.all():
+            raise ValueError(
+                "All objects are marked as outliers. At least one object must not be an outlier."
+            )
+
+    tracks_data = _get_data_or_none(dataset, config.track_column)
+    if tracks_data is None:
+        tracks_data = _get_data_or_none(dataset, config.object_id_column)
```

Comment:
is it worth a warning or some output to tell users what is happening here?

Reply:
Yeah, definitely. I'll update this.

Reply:
I also added an error for if the times column is missing while I was at it + a unit test ✌️